packages/docs/pages/experimental/logging-config.mdx

```diff
@@ -0,0 +1,17 @@
+# Configuring Logging on Namada
+
+##### `NAMADA_LOG`
+The logs of the ledger can be configured by setting the `NAMADA_LOG` environment variable. The default value is `info`. The available values are `info`, `debug`, `warn`, `error`.
+
+##### `NAMADA_LOG_COLOR`
+`NAMADA_LOG_COLOR` can be used to switch off colors (true/false). 
+
+##### `NAMADA_LOG_FMT`
+`NAMADA_LOG_FMT` can be set to full (the default), json or pretty.
+
+#### `NAMADA_LOG_DIR` 
+The `NAMADA_LOG_DIR` env var can be used for logging to file(s) and can be set to a directory path. The default value is empty. If set, the logs will be written to files in the specified directory. The log files will be named `namada.log` and `namada.err.log`. 
+The log files will be rotated daily and the logs will be written to the files in the format `namada.log.YYYY-MM-DD` and `namada.err.log.YYYY-MM-DD`. The log files will be compressed using gzip.
```

Comment:
I think they'll be rotated only with non-default `NAMADA_LOG_ROLLING`

Reply:
Yes, you are right, by default we use ::Never